[2017-05-06 13:16:31] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/1"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/1\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 13:19:10] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/1"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/1\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 13:19:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/1"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/1\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 13:44:34] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 13:49:28] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:27:31] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:27:41] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:34:22] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:34:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /article"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /article\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:34:45] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:44:19] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:45:52] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:47:50] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:47:58] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:48:07] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:48:14] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:49:40] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:50:38] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /articles"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /articles\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:51:11] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/3"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/3\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:51:16] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/1"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/1\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:51:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/2"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/2\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:51:54] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/2"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/2\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:52:50] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/1"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/1\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 14:53:06] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/1"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/1\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 17:21:22] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/3"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/3\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 17:21:33] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/3"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/3\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 17:21:40] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/3"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/3\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 17:59:57] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/3"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/3\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-06 18:00:11] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/1"" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/1\" at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainerUrlMatcher.php:39)"} []
[2017-05-09 09:25:03] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://devsymfony.fr/","method":"GET"} []
[2017-05-09 09:25:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-05-09 09:25:04] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The directory "/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer" is not writable." at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php line 17 {"exception":"[object] (InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17)"} []
[2017-05-09 09:25:04] request.CRITICAL: Exception thrown when handling an exception (InvalidArgumentException: The directory "/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php line 17) {"exception":"[object] (InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17)"} []
[2017-05-09 09:25:04] php.CRITICAL: Uncaught Exception: The directory "/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer" is not writable. {"exception":"[object] (InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17, InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17)"} []
[2017-05-09 09:25:04] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The directory "/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer" is not writable." at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php line 17 {"exception":"[object] (InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17, InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17)"} []
[2017-05-09 09:25:04] request.CRITICAL: Exception thrown when handling an exception (InvalidArgumentException: The directory "/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php line 17) {"exception":"[object] (InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17)"} []
[2017-05-09 09:25:04] php.CRITICAL: Uncaught Exception: The directory "/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer" is not writable. {"exception":"[object] (InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17, InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17, InvalidArgumentException(code: 0): The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable. at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17)"} []
[2017-05-09 09:25:04] php.CRITICAL: Fatal Error: Uncaught exception 'InvalidArgumentException' with message 'The directory "/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer" is not writable.' in /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17 Stack trace: #0 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainer.php(1548): Metadata\Cache\FileCache->__construct('/Volumes/DATA/S...') #1 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php(3534): appProdProjectContainer->getJmsSerializerService() #2 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainer.php(3235): Symfony\Component\DependencyInjection\Container->get('jms_serializer') #3 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php(3534): appProdProjectContainer->getTwigService() #4 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainer.php(2727): Symfony\Component\DependencyInjection\Container->get('twig') #5 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/clas {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Uncaught exception 'InvalidArgumentException' with message 'The directory \"/Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/jms_serializer\" is not writable.' in /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17\nStack trace:\n#0 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainer.php(1548): Metadata\\Cache\\FileCache->__construct('/Volumes/DATA/S...')\n#1 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php(3534): appProdProjectContainer->getJmsSerializerService()\n#2 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainer.php(3235): Symfony\\Component\\DependencyInjection\\Container->get('jms_serializer')\n#3 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/classes.php(3534): appProdProjectContainer->getTwigService()\n#4 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/appProdProjectContainer.php(2727): Symfony\\Component\\DependencyInjection\\Container->get('twig')\n#5 /Volumes/DATA/Site/Symfony/blog-api/var/cache/prod/clas at /Volumes/DATA/Site/Symfony/blog-api/vendor/jms/metadata/src/Metadata/Cache/FileCache.php:17)"} []
[2017-05-12 17:49:23] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:94)"} []
[2017-05-12 18:24:59] request.INFO: Matched route "app_article_list". {"route":"app_article_list","route_parameters":{"_controller":"AppBundle\\Controller\\ArticleController::listAction","_route":"app_article_list"},"request_uri":"http://devbackrestsymfony.fr/articles","method":"GET"} []
[2017-05-12 18:24:59] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-05-12 18:24:59] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "No @ParamInterface configuration for parameter 'page'." at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/friendsofsymfony/rest-bundle/Request/ParamFetcher.php line 98 {"exception":"[object] (InvalidArgumentException(code: 0): No @ParamInterface configuration for parameter 'page'. at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/friendsofsymfony/rest-bundle/Request/ParamFetcher.php:98)"} []
[2017-05-12 18:25:30] request.INFO: Matched route "app_article_list". {"route":"app_article_list","route_parameters":{"_controller":"AppBundle\\Controller\\ArticleController::listAction","_route":"app_article_list"},"request_uri":"http://devbackrestsymfony.fr/articles","method":"GET"} []
[2017-05-12 18:25:30] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-05-12 18:25:30] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "No @ParamInterface configuration for parameter 'page'." at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/friendsofsymfony/rest-bundle/Request/ParamFetcher.php line 98 {"exception":"[object] (InvalidArgumentException(code: 0): No @ParamInterface configuration for parameter 'page'. at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/friendsofsymfony/rest-bundle/Request/ParamFetcher.php:98)"} []
[2017-05-12 18:25:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles/"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles/\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:94)"} []
[2017-05-12 18:25:34] request.INFO: Matched route "app_article_list". {"route":"app_article_list","route_parameters":{"_controller":"AppBundle\\Controller\\ArticleController::listAction","_route":"app_article_list"},"request_uri":"http://devbackrestsymfony.fr/articles","method":"GET"} []
[2017-05-12 18:25:34] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-05-12 18:25:34] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "No @ParamInterface configuration for parameter 'page'." at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/friendsofsymfony/rest-bundle/Request/ParamFetcher.php line 98 {"exception":"[object] (InvalidArgumentException(code: 0): No @ParamInterface configuration for parameter 'page'. at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/friendsofsymfony/rest-bundle/Request/ParamFetcher.php:98)"} []
[2017-05-12 18:28:16] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /articles&limit=3"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /articles&limit=3\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:105)"} []
[2017-05-12 18:31:34] request.INFO: Matched route "app_article_list". {"route":"app_article_list","route_parameters":{"_controller":"AppBundle\\Controller\\ArticleController::listAction","_route":"app_article_list"},"request_uri":"http://devbackrestsymfony.fr/articles?limit=3","method":"GET"} []
[2017-05-12 18:31:34] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-05-12 18:31:34] request.CRITICAL: Uncaught PHP Exception ReflectionException: "Property AppBundle\Entity\Author::$articles does not exist" at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/jms/metadata/src/Metadata/PropertyMetadata.php line 75 {"exception":"[object] (ReflectionException(code: 0): Property AppBundle\\Entity\\Author::$articles does not exist at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/jms/metadata/src/Metadata/PropertyMetadata.php:75)"} []
[2017-05-12 19:24:50] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:105)"} []
[2017-05-12 19:28:23] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:105)"} []
[2017-05-12 19:28:38] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:105)"} []
[2017-05-12 19:28:43] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:105)"} []
[2017-05-12 19:29:04] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:105)"} []
[2017-05-12 19:29:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:105)"} []
[2017-05-13 12:41:13] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev.phphttp://devbackrestsymfony.fr/articles/1"" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php line 3941 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev.phphttp://devbackrestsymfony.fr/articles/1\" at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/classes.php:3941, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Volumes/DATA/Site/Symfony/blog-api/back/var/cache/prod/appProdProjectContainerUrlMatcher.php:105)"} []
[2017-05-13 12:42:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotAcceptableHttpException: "No matching accepted Response format could be determined" at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/friendsofsymfony/rest-bundle/EventListener/FormatListener.php line 71 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotAcceptableHttpException(code: 0): No matching accepted Response format could be determined at /Volumes/DATA/Site/Symfony/blog-api/back/vendor/friendsofsymfony/rest-bundle/EventListener/FormatListener.php:71)"} []
